packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -2035,6 +2046,14 @@ class InputDecoration {
   /// See [Icon], [ImageIcon].
   final Widget prefixIcon;
 
+  /// Optional widget to place on the line before the input.
```

Comment:
It's worth explaining that the prefix widget's baseline will be lined up with the input baseline.